services/server/server.go

```diff
@@ -522,6 +522,13 @@ func (s *Server) validateChangeSettingsRequest(ctx context.Context, req *serverp
 
 	// check request parameters compatibility with environment variables
 
+	if req.DisableUpdates && !s.envSettings.DisableUpdates {
+		return status.Error(codes.FailedPrecondition, "Updates are enabled via DISABLE_UPDATES environment variable.")
+	}
```

Comment:
let's remove it. `s.envSettings.DisableUpdates` is false by default, so users won't be able to disable updates.